services.py

```diff
@@ -88,42 +76,80 @@ def get_param_from_config(self, config_path: str, param_name: str):
             self.logger.error(f'Config file not found: {config_path}')
             return None
 
-    def get_lat_lon(self):
-        g = geocoder.ip('me')
-        return g.latlng
+    def formatDegreesMinutes(self, coordinates, digits):
+    
+        parts = coordinates.split(".")
+
+        if (len(parts) != 2):
+            return coordinates
+
+        if (digits > 3 or digits < 2):
+            return coordinates
```

Comment:
This looks super arbitrary, you need to explain why. Also `if digits not in (2, 3):`

Reply:
I didn't write this code, it was one of the ones that were given to me by the employer (I guess they want it that way?)